Tests/test_NCBI_qblast.py

```diff
@@ -48,15 +48,16 @@ def test_blastp_nr_actin(self):
         # GI:160837788 aka NP_075631.2
         # the actin related protein 2/3 complex, subunit 1B [Mus musculus]
         self.run_qblast("blastp", "nr", "NP_075631.2", 0.001,
-                        "rat [ORGN]", ['9506405', '13592137', '37589612', '149064087', '56912225'])
+                        "rat [ORGN]", dict(megablast='FALSE'), ['9506405', '13592137', '37589612', '149064087', '56912225'])
```

Comment:
Style wise, avoid the very long line by using more line breaks.

Also why not write ``{'megablast': 'FALSE'}`` here (and likewise later on)?

Reply:
No special reason. I'll change it to `{'megablast': 'FALSE'}` then